python/utils.py

```diff
@@ -278,14 +278,24 @@ def override_build_config_from_env(config):
         config.version_tag = os.getenv("NVFUSER_BUILD_VERSION_TAG")
 
 
+def get_default_install_prefix():
+    cwd = os.path.dirname(os.path.abspath(__file__))
+    return os.path.join(cwd, "nvfuser_common")
+
+
 class build_ext(setuptools.command.build_ext.build_ext):
+    def __init__(self, *args, **kwargs):
+        install_dir = kwargs.pop("install_dir", "")
+        self.install_dir = install_dir if install_dir else get_default_install_prefix()
+        super().__init__(*args, **kwargs)
+
     def copy_library(self, ext, library_name):
         # Copy files on necessity.
         filename = self.get_ext_filename(self.get_ext_fullname(ext.name))
         fileext = os.path.splitext(filename)[1]
 
         libnvfuser_path = os.path.join(
-            "./nvfuser_common/lib", f"{library_name}{fileext}"
+            os.path.join(self.install_dir, "lib"), f"libnvfuser{fileext}"
```

Comment:
@rdspring1 is it a mistake to hardcode `libnvfuser` here? Note library_name can be `libnvfuser_next`.

Reply:
Yes. It should be `{library_name}` because we create multiple python extensions.